### ISSUE TITLE
"No plan found in TAP output"

### ISSUE THREAD

**Issue by bortzmeyer** [dotse/zonemaster#438]
Cannot install Zonemaster this morning:

% perl -MCPAN -e 'install Zonemaster'        
Reading '/home/stephane/.cpan/Metadata'
  Database was generated on Tue, 08 Dec 2015 08:29:02 GMT
Running install for module 'Zonemaster'
Checksum for /home/stephane/.cpan/sources/authors/id/I/II/IIS/Zonemaster-v1.0.10.tar.gz ok
Configuring I/II/IIS/Zonemaster-v1.0.10.tar.gz with Makefile.PL
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Zonemaster
Writing MYMETA.yml and MYMETA.json
  IIS/Zonemaster-v1.0.10.tar.gz
  /usr/bin/perl Makefile.PL INSTALLDIRS=site -- OK
Running make for I/II/IIS/Zonemaster-v1.0.10.tar.gz
cp lib/Zonemaster/Constants.pm blib/lib/Zonemaster/Constants.pm
cp lib/Zonemaster/Logger/Entry.pm blib/lib/Zonemaster/Logger/Entry.pm
cp lib/Zonemaster/DNSName.pm blib/lib/Zonemaster/DNSName.pm
cp lib/Zonemaster/Util.pm blib/lib/Zonemaster/Util.pm
cp lib/Zonemaster/Config.pm blib/lib/Zonemaster/Config.pm
cp lib/Zonemaster/Translator.pm blib/lib/Zonemaster/Translator.pm
cp lib/Zonemaster/TestMethods.pm blib/lib/Zonemaster/TestMethods.pm
cp lib/Zonemaster/Recursor.pm blib/lib/Zonemaster/Recursor.pm
cp lib/Zonemaster/Test/DNSSEC.pm blib/lib/Zonemaster/Test/DNSSEC.pm
cp lib/Zonemaster/NSArray.pm blib/lib/Zonemaster/NSArray.pm
cp lib/Zonemaster/Test/Delegation.pm blib/lib/Zonemaster/Test/Delegation.pm
cp lib/Zonemaster/Test/Consistency.pm blib/lib/Zonemaster/Test/Consistency.pm
cp lib/Zonemaster/Test/Connectivity.pm blib/lib/Zonemaster/Test/Connectivity.pm
cp lib/Zonemaster/Test.pm blib/lib/Zonemaster/Test.pm
cp lib/Zonemaster/Nameserver.pm blib/lib/Zonemaster/Nameserver.pm
cp lib/Zonemaster/Logger.pm blib/lib/Zonemaster/Logger.pm
cp lib/Zonemaster/Exception.pm blib/lib/Zonemaster/Exception.pm
cp lib/Zonemaster/Test/Basic.pm blib/lib/Zonemaster/Test/Basic.pm
cp lib/Zonemaster/Packet.pm blib/lib/Zonemaster/Packet.pm
cp lib/Zonemaster/ASNLookup.pm blib/lib/Zonemaster/ASNLookup.pm
cp lib/Zonemaster/Overview.pod blib/lib/Zonemaster/Overview.pod
cp lib/Zonemaster/Test/Syntax.pm blib/lib/Zonemaster/Test/Syntax.pm
cp lib/Zonemaster/Test/Zone.pm blib/lib/Zonemaster/Test/Zone.pm
cp lib/Zonemaster/Zone.pm blib/lib/Zonemaster/Zone.pm
cp lib/Zonemaster/Nameserver/Cache.pm blib/lib/Zonemaster/Nameserver/Cache.pm
cp lib/Zonemaster/Test/Address.pm blib/lib/Zonemaster/Test/Address.pm
cp lib/Zonemaster.pm blib/lib/Zonemaster.pm
cp lib/Zonemaster/Test/Nameserver.pm blib/lib/Zonemaster/Test/Nameserver.pm
cp lib/Zonemaster/Test/Example.pm blib/lib/Zonemaster/Test/Example.pm
Manifying blib/man3/Zonemaster.3pm
Manifying blib/man3/Zonemaster::ASNLookup.3pm
Manifying blib/man3/Zonemaster::Config.3pm
Manifying blib/man3/Zonemaster::Constants.3pm
Manifying blib/man3/Zonemaster::DNSName.3pm
Manifying blib/man3/Zonemaster::Exception.3pm
Manifying blib/man3/Zonemaster::Logger.3pm
Manifying blib/man3/Zonemaster::Logger::Entry.3pm
Manifying blib/man3/Zonemaster::NSArray.3pm
Manifying blib/man3/Zonemaster::Nameserver.3pm
Manifying blib/man3/Zonemaster::Nameserver::Cache.3pm
Manifying blib/man3/Zonemaster::Overview.3pm
Manifying blib/man3/Zonemaster::Packet.3pm
Manifying blib/man3/Zonemaster::Recursor.3pm
Manifying blib/man3/Zonemaster::Test.3pm
Manifying blib/man3/Zonemaster::Test::Address.3pm
Manifying blib/man3/Zonemaster::Test::Basic.3pm
Manifying blib/man3/Zonemaster::Test::Connectivity.3pm
Manifying blib/man3/Zonemaster::Test::Consistency.3pm
Manifying blib/man3/Zonemaster::Test::DNSSEC.3pm
Manifying blib/man3/Zonemaster::Test::Delegation.3pm
Manifying blib/man3/Zonemaster::Test::Example.3pm
Manifying blib/man3/Zonemaster::Test::Nameserver.3pm
Manifying blib/man3/Zonemaster::Test::Syntax.3pm
Manifying blib/man3/Zonemaster::Test::Zone.3pm
Manifying blib/man3/Zonemaster::TestMethods.3pm
Manifying blib/man3/Zonemaster::Translator.3pm
Manifying blib/man3/Zonemaster::Util.3pm
Manifying blib/man3/Zonemaster::Zone.3pm
  IIS/Zonemaster-v1.0.10.tar.gz
  /usr/bin/make -- OK
Running make test
PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'inc', 'blib/lib', 'blib/arch')" t/*.t
t/00-load.t ............ 1/? # Testing Zonemaster v1.0.10, Perl 5.020002, /usr/bin/perl
t/00-load.t ............ ok   
t/asn.t ................ ok   
t/config.t ............. ok    
t/dnsname.t ............ ok    
t/logger.t ............. ok    
t/nameserver-axfr.t .... ok   
t/nameserver.t ......... ok    
t/old-bugs.t ........... ok    
t/pod-coverage.t ....... 1/? # no public symbols defined for Zonemaster::Constants
# no public symbols defined for Zonemaster::NSArray
t/pod-coverage.t ....... ok    
t/recursor.t ........... ok    
t/Test-address.t ....... ok    
t/Test-basic.t ......... ok    
t/Test-connectivity.t .. ok    
t/Test-consistency.t ... ok    
t/Test-delegation.t .... ok    
t/Test-dnssec-more.t ... ok    
t/Test-dnssec.t ........ ok     
t/Test-nameserver.t .... All 34 subtests passed 
	(less 1 skipped subtest: 33 okay)
t/Test-syntax.t ........ ok    
t/Test-zone.t .......... ok    
t/translator.t ......... ok   
t/undelegated.t ........ ok   
t/util.t ............... ok   
t/zone.t ............... ok     
t/zonemaster.t ......... ok    

Test Summary Report
-------------------
t/Test-nameserver.t  (Wstat: 13 Tests: 34 Failed: 0)
  Non-zero wait status: 13
  Parse errors: No plan found in TAP output
Files=25, Tests=825, 222 wallclock secs ( 0.19 usr  0.03 sys + 205.74 cusr  0.99 csys = 206.95 CPU)
Result: FAIL
Failed 1/25 test programs. 0/825 subtests failed.
Makefile:918: recipe for target 'test_dynamic' failed
make: *** [test_dynamic] Error 255
  IIS/Zonemaster-v1.0.10.tar.gz
  /usr/bin/make test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports IIS/Zonemaster-v1.0.10.tar.gz

**Comment by sandoche2k** [comment]
Works for me :

 perl -MCPAN -e 'install Zonemaster'

CPAN.pm requires configuration, but most of it can be done automatically.
If you answer 'no' below, you will enter an interactive dialog for each
configuration option instead.

Would you like to configure as much as possible automatically? [yes] yes

 <install_help>

Warning: You do not have write permission for Perl library directories.

To install modules, you need to configure a local Perl library directory or
escalate your privileges.  CPAN can help you by bootstrapping the local::lib
module or by configuring itself to use 'sudo' (if available).  You may also
resolve this problem manually if you need to customize your setup.

What approach do you want?  (Choose 'local::lib', 'sudo' or 'manual')
 [local::lib]

Autoconfigured everything but 'urllist'.

Now you need to choose your CPAN mirror sites.  You can let me
pick mirrors for you, you can select them from a list or you
can enter them by hand.

Would you like me to automatically choose some CPAN mirror
sites for you? (This means connecting to the Internet) [yes] yes
Trying to fetch a mirror list from the Internet
Fetching with LWP:
http://www.perl.org/CPAN/MIRRORED.BY

Looking for CPAN mirrors near you (please be patient)
......................... done!

New urllist
  http://www.pirbot.com/mirrors/cpan/
  http://mirror.ox.ac.uk/sites/www.cpan.org/
  http://ftp.perl.se/

Autoconfiguration complete.

Attempting to bootstrap local::lib...

Writing /home/sandoche/.cpan/CPAN/MyConfig.pm for bootstrap...
commit: wrote '/home/sandoche/.cpan/CPAN/MyConfig.pm'
Fetching with LWP:
http://www.pirbot.com/mirrors/cpan/authors/01mailrc.txt.gz
Going to read '/home/sandoche/.cpan/sources/authors/01mailrc.txt.gz'
............................................................................DONE
Fetching with LWP:
http://www.pirbot.com/mirrors/cpan/modules/02packages.details.txt.gz
Going to read
'/home/sandoche/.cpan/sources/modules/02packages.details.txt.gz'
  Database was generated on Mon, 07 Dec 2015 21:41:02 GMT
..............
  New CPAN.pm version (v2.10) available.
  [Currently running version is v1.960001]
  You might want to try
    install CPAN
    reload cpan
  to both upgrade CPAN.pm and run the new version without leaving
  the current session.


..............................................................DONE
Fetching with LWP:
http://www.pirbot.com/mirrors/cpan/modules/03modlist.data.gz
Going to read '/home/sandoche/.cpan/sources/modules/03modlist.data.gz'
DONE
Going to write /home/sandoche/.cpan/Metadata
Running make for H/HA/HAARG/local-lib-2.000018.tar.gz
Fetching with LWP:
http://www.pirbot.com/mirrors/cpan/authors/id/H/HA/HAARG/local-lib-2.000018.tar.gz
Fetching with LWP:
http://www.pirbot.com/mirrors/cpan/authors/id/H/HA/HAARG/CHECKSUMS
Checksum for
/home/sandoche/.cpan/sources/authors/id/H/HA/HAARG/local-lib-2.000018.tar.gz
ok

  CPAN.pm: Going to build H/HA/HAARG/local-lib-2.000018.tar.gz

Attempting to create directory /home/sandoche/perl5

Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for local::lib
Writing MYMETA.yml and MYMETA.json
cp lib/POD2/PT_BR/local/lib.pod blib/lib/POD2/PT_BR/local/lib.pod
cp lib/lib/core/only.pm blib/lib/lib/core/only.pm
cp lib/local/lib.pm blib/lib/local/lib.pm
cp lib/POD2/DE/local/lib.pod blib/lib/POD2/DE/local/lib.pod
Manifying 4 pod documents
  HAARG/local-lib-2.000018.tar.gz
  /usr/bin/make -- OK
Running make test
PERL_DL_NONLAZY=1 "/usr/bin/perl" -I/home/sandoche/perl5/lib/perl5
"-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef
*Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/bad_variables.t ...... ok
t/carp-mismatch.t ...... ok
t/classmethod.t ........ ok
t/coderefs_in_inc.t .... ok
t/de-dup.t ............. ok
t/lib-core-only.t ...... ok
t/pipeline.t ........... ok
t/shell.t .............. ok
t/stackable.t .......... ok
t/subroutine-in-inc.t .. ok
t/taint-mode.t ......... ok
All tests successful.
Files=11, Tests=125,  1 wallclock secs ( 0.02 usr  0.04 sys +  0.68 cusr
0.14 csys =  0.88 CPU)
Result: PASS
  HAARG/local-lib-2.000018.tar.gz
  /usr/bin/make test -- OK
Running make install
Manifying 4 pod documents
Installing /home/sandoche/perl5/lib/perl5/lib/core/only.pm
Installing /home/sandoche/perl5/lib/perl5/POD2/PT_BR/local/lib.pod
Installing /home/sandoche/perl5/lib/perl5/POD2/DE/local/lib.pod
Installing /home/sandoche/perl5/lib/perl5/local/lib.pm
Installing /home/sandoche/perl5/man/man3/local::lib.3pm
Installing /home/sandoche/perl5/man/man3/POD2::PT_BR::local::lib.3pm
Installing /home/sandoche/perl5/man/man3/POD2::DE::local::lib.3pm
Installing /home/sandoche/perl5/man/man3/lib::core::only.3pm
Appending installation info to
/home/sandoche/perl5/lib/perl5/x86_64-linux-gnu-thread-multi/perllocal.pod
  HAARG/local-lib-2.000018.tar.gz
  /usr/bin/make install  -- OK

local::lib is installed. You must now add the following environment
variables
to your shell configuration files (or registry, if you are on Windows) and
then restart your command line shell and CPAN before installing modules:

PATH="/home/sandoche/perl5/bin${PATH+:}${PATH}"; export PATH;
PERL5LIB="/home/sandoche/perl5/lib/perl5${PERL5LIB+:}${PERL5LIB}"; export
PERL5LIB;
PERL_LOCAL_LIB_ROOT="/home/sandoche/perl5${PERL_LOCAL_LIB_ROOT+:}${PERL_LOCAL_LIB_ROOT}";
export PERL_LOCAL_LIB_ROOT;
PERL_MB_OPT="--install_base \"/home/sandoche/perl5\""; export PERL_MB_OPT;
PERL_MM_OPT="INSTALL_BASE=/home/sandoche/perl5"; export PERL_MM_OPT;

Would you like me to append that to /home/sandoche/.bashrc now? [yes] yes


commit: wrote '/home/sandoche/.cpan/CPAN/MyConfig.pm'

You can re-run configuration any time with 'o conf init' in the CPAN shell
Running install for module 'Zonemaster'
Running make for I/II/IIS/Zonemaster-v1.0.10.tar.gz
Fetching with LWP:
http://www.pirbot.com/mirrors/cpan/authors/id/I/II/IIS/Zonemaster-v1.0.10.tar.gz
Fetching with LWP:
http://www.pirbot.com/mirrors/cpan/authors/id/I/II/IIS/CHECKSUMS
Checksum for
/home/sandoche/.cpan/sources/authors/id/I/II/IIS/Zonemaster-v1.0.10.tar.gz
ok

  CPAN.pm: Going to build I/II/IIS/Zonemaster-v1.0.10.tar.gz

Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Zonemaster
Writing MYMETA.yml and MYMETA.json
cp lib/Zonemaster/TestMethods.pm blib/lib/Zonemaster/TestMethods.pm
cp lib/Zonemaster/Nameserver.pm blib/lib/Zonemaster/Nameserver.pm
cp lib/Zonemaster/Test/Connectivity.pm
blib/lib/Zonemaster/Test/Connectivity.pm
cp lib/Zonemaster/Test/Syntax.pm blib/lib/Zonemaster/Test/Syntax.pm
cp lib/Zonemaster/Test/Example.pm blib/lib/Zonemaster/Test/Example.pm
cp lib/Zonemaster/Logger.pm blib/lib/Zonemaster/Logger.pm
cp lib/Zonemaster/Translator.pm blib/lib/Zonemaster/Translator.pm
cp lib/Zonemaster/Test/Basic.pm blib/lib/Zonemaster/Test/Basic.pm
cp lib/Zonemaster/Test/Nameserver.pm blib/lib/Zonemaster/Test/Nameserver.pm
cp lib/Zonemaster/NSArray.pm blib/lib/Zonemaster/NSArray.pm
cp lib/Zonemaster/Exception.pm blib/lib/Zonemaster/Exception.pm
cp lib/Zonemaster/Test/Delegation.pm blib/lib/Zonemaster/Test/Delegation.pm
cp lib/Zonemaster/Util.pm blib/lib/Zonemaster/Util.pm
cp lib/Zonemaster/Test/DNSSEC.pm blib/lib/Zonemaster/Test/DNSSEC.pm
cp lib/Zonemaster/DNSName.pm blib/lib/Zonemaster/DNSName.pm
cp lib/Zonemaster/Packet.pm blib/lib/Zonemaster/Packet.pm
cp lib/Zonemaster/Zone.pm blib/lib/Zonemaster/Zone.pm
cp lib/Zonemaster/Recursor.pm blib/lib/Zonemaster/Recursor.pm
cp lib/Zonemaster/Nameserver/Cache.pm
blib/lib/Zonemaster/Nameserver/Cache.pm
cp lib/Zonemaster/Overview.pod blib/lib/Zonemaster/Overview.pod
cp lib/Zonemaster/Test/Address.pm blib/lib/Zonemaster/Test/Address.pm
cp lib/Zonemaster/Test/Consistency.pm
blib/lib/Zonemaster/Test/Consistency.pm
cp lib/Zonemaster/Config.pm blib/lib/Zonemaster/Config.pm
cp lib/Zonemaster/ASNLookup.pm blib/lib/Zonemaster/ASNLookup.pm
cp lib/Zonemaster/Test.pm blib/lib/Zonemaster/Test.pm
cp lib/Zonemaster.pm blib/lib/Zonemaster.pm
cp lib/Zonemaster/Test/Zone.pm blib/lib/Zonemaster/Test/Zone.pm
cp lib/Zonemaster/Constants.pm blib/lib/Zonemaster/Constants.pm
cp lib/Zonemaster/Logger/Entry.pm blib/lib/Zonemaster/Logger/Entry.pm
Manifying 29 pod documents
  IIS/Zonemaster-v1.0.10.tar.gz
  /usr/bin/make -- OK
Running make test
PERL_DL_NONLAZY=1 "/usr/bin/perl" "-MExtUtils::Command::MM"
"-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0,
'inc', 'blib/lib', 'blib/arch')" t/*.t
t/00-load.t ............ 1/? # Testing Zonemaster v1.0.10, Perl 5.014002,
/usr/bin/perl
t/00-load.t ............ ok
t/asn.t ................ ok
t/config.t ............. ok
t/dnsname.t ............ ok
t/logger.t ............. ok
t/nameserver-axfr.t .... ok
t/nameserver.t ......... ok
t/old-bugs.t ........... ok
t/pod-coverage.t ....... skipped: Need Pod::Coverage to run these tests.
t/recursor.t ........... ok
t/Test-address.t ....... ok
t/Test-basic.t ......... ok
t/Test-connectivity.t .. ok
t/Test-consistency.t ... ok
t/Test-delegation.t .... ok
t/Test-dnssec-more.t ... ok
t/Test-dnssec.t ........ ok
t/Test-nameserver.t .... ok
t/Test-syntax.t ........ ok
t/Test-zone.t .......... ok
t/translator.t ......... ok
t/undelegated.t ........ ok
t/util.t ............... ok
t/zone.t ............... ok
t/zonemaster.t ......... ok
All tests successful.
Files=25, Tests=827, 155 wallclock secs ( 0.09 usr  0.10 sys + 136.42 cusr
3.47 csys = 140.08 CPU)
Result: PASS
  IIS/Zonemaster-v1.0.10.tar.gz
  /usr/bin/make test -- OK
Running make install
Manifying 29 pod documents
Installing /home/sandoche/perl5/lib/perl5/Zonemaster.pm
Installing /home/sandoche/perl5/lib/perl5/auto/share/dist/Zonemaster/sv.po
Installing /home/sandoche/perl5/lib/perl5/auto/share/dist/Zonemaster/en.po
Installing
/home/sandoche/perl5/lib/perl5/auto/share/dist/Zonemaster/Makefile
Installing /home/sandoche/perl5/lib/perl5/auto/share/dist/Zonemaster/fr.po
Installing
/home/sandoche/perl5/lib/perl5/auto/share/dist/Zonemaster/config.json
Installing
/home/sandoche/perl5/lib/perl5/auto/share/dist/Zonemaster/policy.json
Installing
/home/sandoche/perl5/lib/perl5/auto/share/dist/Zonemaster/fr_FR.UTF-8.po
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/ASNLookup.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Util.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/DNSName.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/NSArray.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Logger.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Recursor.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Constants.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Zone.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Packet.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Translator.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Overview.pod
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Config.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/TestMethods.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Nameserver.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Exception.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Nameserver/Cache.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Logger/Entry.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Delegation.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Zone.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/DNSSEC.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Syntax.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Basic.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Connectivity.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Consistency.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Address.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Nameserver.pm
Installing /home/sandoche/perl5/lib/perl5/Zonemaster/Test/Example.pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Nameserver::Cache.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::ASNLookup.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Recursor.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Exception.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Connectivity.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Packet.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Nameserver.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Syntax.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Address.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Logger::Entry.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Logger.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Delegation.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Zone.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::DNSName.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Constants.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Util.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Config.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::DNSSEC.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Basic.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::TestMethods.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Zone.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Consistency.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::NSArray.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Example.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Test::Nameserver.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Overview.3pm
Installing /home/sandoche/perl5/man/man3/Zonemaster::Translator.3pm
Appending installation info to
/home/sandoche/perl5/lib/perl5/x86_64-linux-gnu-thread-multi/perllocal.pod
  IIS/Zonemaster-v1.0.10.tar.gz
  /usr/bin/make install  -- OK

**Comment by bortzmeyer** [comment]
On Tue, Dec 08, 2015 at 01:33:31AM -0800,
 sandoche2k <notifications@github.com> wrote 
 a message of 1125 lines which said:

> CPAN.pm requires configuration, but most of it can be done
> automatically.

I know :-) It worked before on this machine (this is how Zonemaster is
currently installed, I wanted to upgrade).

**Comment by sandoche2k** [comment]
An upgrade from version 1.0.8 to 1.0.10 using CPAN worked for me in Ubuntu
14.0.4.

**Comment by sandoche2k** [comment]
@vlevigneron we are having this issue in install. Checked today on my Ubuntu. Can you have a look?

**Comment by vlevigneron** [comment]
Is it still an issue ? I can not reproduce it with the last release...

**Comment by mtoma** [comment]
I can't reproduce it either anymore.
As far as I'm concerned the issue can be closed.

----- Original Message -----

**Comment by pawal** [comment]
Any of you know what fixed it? Closing...

**Comment by mtoma** [comment]
I can't tell exactly. I had to upgrade my machine since I first saw this problem so I didn't have the possibility to test this in a stable environment.
My bet is that we are missing a minimum version constraint on some of the module bundling packages.
This should be easily spotted by the CPAN testers network. If Zonemaster has 100% pass score than we can ignore this problem completely.

----- Original Message -----
